packages/core/src/PackageContext.ts

```diff
@@ -63,7 +56,7 @@ export class PackageContext implements Context {
     }
   }
 
-  public addError({ file, message, longMessage, fixer }: FailureOptions) {
+  public addError({ file, message, longMessage, fixer }: AddErrorOptions) {
```

Comment:
@ericanderson I may have misunderstood the intention of this class. Instead of updating the typings to require `file`, should this function be updated to handle `file === undefined`?